### PR TITLE
Trusted include

### PR DESCRIPTION
**Pull request by maceq687**
Created trusted-section jekyll include and added it to services layout

### CODE REVIEW

File: _includes/trusted.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+</style>
+
+<section class="trusted-section">
+  <div class="trusted-container">
+    <h4>deranged is trusted by:</h4>
+    <a href="/cases" class="logo-container">
+      <img src="/img/marmar.png" class="client-logo" alt="" />
+      <img src="/img/fischer-lighting.png" class="client-logo" alt="" />
+      <img src="/img/kbh-stadsarkiv.png" class="client-logo" alt="" />
+      <img src="/img/nornorm-km.png" class="client-logo" alt="" />
```
Comment (hypesystem) [inline _includes/trusted.html]: The non-png images we could probably improve performance on by using an img srcset and doing webp versions for browsers that support it? Also differenty sizes etc.

File: _includes/trusted.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
+<style type="text/css" scoped>
+  .trusted-section {
+    color: #eaf1f7;
+    background-color: #2c3e50;
+    padding: 4em;
+  }
+
+  .trusted-container {
```
Comment (hypesystem) [inline _includes/trusted.html]: In BEM you would call this `trusted-section__container` to denote the significant relationship

File: _includes/trusted.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    max-width: 1600px;
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 1.5em;
+  }
+
+  .logo-container {
```
Comment (hypesystem) [inline _includes/trusted.html]: `trusted-section__logo-container`

File: _includes/trusted.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    justify-content: space-around;
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  .client-logo {
```
Comment (hypesystem) [inline _includes/trusted.html]: `trusted-section__logo`

File: _includes/trusted.html [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    margin-right: 12px;
+    max-height: 68px;
+    padding: 8px;
+  }
+
+  .client-logo.svg {
+    padding: 16px 8px;
```
Comment (hypesystem) [inline _includes/trusted.html]: This seems odd... what is the reason for this? Isn't it random that the SVG happens to not have enough padding, or? Why is `svg` the significant class name here?

**Review by hypesystem — APPROVED** [review]

File: _includes/trusted.html (hunk 2 above)
Comment (maceq687) [reply _includes/trusted.html]: Fixed

File: _includes/trusted.html (hunk 3 above)
Comment (maceq687) [reply _includes/trusted.html]: Fixed

File: _includes/trusted.html (hunk 4 above)
Comment (maceq687) [reply _includes/trusted.html]: Fixed

File: _includes/trusted.html (hunk 5 above)
Comment (maceq687) [reply _includes/trusted.html]: That was my dirty workaround for this specific svg that has a different padding that the remaining pngs. Proper way would be to make it in the same format (png) and with the same padding as the rest of the images here have.

Comment (maceq687) [reply _includes/trusted.html]: Fixed

**Pull request merged**